apps/address-book/app/components/Card/Empty.js

```diff
@@ -41,11 +56,11 @@ Empty.propTypes = {
 }
 
 const EmptyWrapper = styled.div`
-  ${unselectable};
```

Comment:
Note that I also got rid of `unselectable`. I have tried to select this sort of text multiple times while troubleshooting, and it's always so annoying that I can't. I don't see any benefit to making these cards unselectable.

This change is not related, other than that I was modifying & alphabetizing all the other styles here.